src/Symfony/Component/DependencyInjection/ContainerBuilder.php

```diff
@@ -407,6 +409,28 @@ public function fileExists(string $path, $trackContents = true): bool
         return $exists;
     }
 
+    final public function willBeAvailable(string $package, string $class, array $parentPackages): bool
+    {
+        if (!class_exists($class) && !interface_exists($class, false) && !trait_exists($class, false)) {
+            return false;
+        }
+
+        if (class_exists(FullStack::class) || !class_exists(InstalledVersions::class) || !InstalledVersions::isInstalled($package) || InstalledVersions::isInstalled($package, true)) {
```

Comment:
Regarding `FullStack::class`, this should not be done in that class. Otherwise, this API has to become an internal one usable only for packages belonging to the fullstack repo instead of making it usable by other bundles too.

Reply:
I needed it in a previous iteration, not anymore. Removed.